Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts as build-stage
+
+# Install deps
+RUN apt-get update -y \
+    && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get install --no-install-recommends -y wget=1.* \
+    && rm -rf /var/lib/apt/lists/*
+RUN wget --progress=dot:giga -O nvim.appimage \
+  'https://github.com/neovim/neovim/releases/download/v0.4.4/nvim.appimage'
+RUN chmod +x nvim.appimage
+RUN ./nvim.appimage --appimage-extract
+RUN chmod -R a+rx /squashfs-root/
+RUN mkdir -p /usr/local/bin
+RUN ln -s /squashfs-root/AppRun /usr/local/bin/nvim
+
+# Drop privileges and build firenvim
+RUN useradd --create-home --user-group user
+USER user
+COPY --chown=user:user . /firenvim
+WORKDIR /firenvim
+RUN npm install
+RUN npm run build
+RUN npm run install_manifests
```

Comment:
AFAIU this installs the manifest in the docker image, which means it won't be available to the browser. The browser expects the manifest to be on the user's computer, not in the docker image.

Reply:
Yes, this will install it in the image, but the lower part copies it to the host if you use the docker command I mentioned in the docs. 

BTW, that command requires Docker BuildKit which requires Docker 18.09 or later. It's possible to support older versions too, but I don't think it's needed (18.09 is already more than 2.5 years old). If you think differently, let me know and I'll support older versions too with a more clunky command.